### ISSUE TITLE
164 report_sched test failure

### ISSUE THREAD

**Issue by honggyukim** [namhyung/uftrace#1555]
164 report_sched test randomly fails as follows.
```
$ ./runtest.py -d 164
Start 1 tests without worker pool
Compiler                  gcc                            clang
Test case                 pg             finstrument-fu  pg             finstrument-fu
------------------------: O0 O1 O2 O3 Os O0 O1 O2 O3 Os  O0 O1 O2 O3 Os O0 O1 O2 O3 Os
t164_report_sched: diff result of -finstrument-functions -O0
--- expect      2022-08-21 22:01:11.049446380 +0100
+++ result      2022-08-21 22:01:11.049446380 +0100
@@ -2,4 +2,4 @@
 1 bar
+2 linux:schedule
 1 usleep
-1 linux:schedule
 2 foo

164 report_sched        : OK OK OK OK OK OK OK OK OK OK  OK OK OK OK OK NG OK OK OK OK
```
It looks because another pre-empted schedule event is captured during test. It can be filtered by #1440 now, but old system does not support distinguishing pre-empted schedule events.

**Comment by honggyukim** [comment]
This can be fixed by #1556.

**Comment by namhyung** [comment]
Then I think we should add `--no-sched-preempt` like below:

```diff
diff --git a/tests/t164_report_sched.py b/tests/t164_report_sched.py
index da48e1d416..4606227c33 100644
--- a/tests/t164_report_sched.py
+++ b/tests/t164_report_sched.py
@@ -34,7 +34,7 @@ from runtest import TestBase
 
     def setup(self):
         self.subcmd = 'report'
-        self.option = '-E linux:schedule'
+        self.option = '-E linux:schedule --no-sched-preempt'
 
     def runcmd(self):
         cmd = TestBase.runcmd(self)
```
